compiler/il/ILOpCodeProperties.hpp

```diff
@@ -8864,6 +8864,21 @@
    },
 
    {
+   /* .opcode               = */ TR::UnsignedOverflowCHK,
+   /* .name                 = */ "UnsignedOverflowCHK",
+   /* .properties1          = */ ILProp1::TreeTop | ILProp1::HasSymbolRef,
+   /* .properties2          = */ ILProp2::Check | ILProp2::MayUseVMThread | ILProp2::MayUseSystemStack,
```

Comment:
if we're using an exception edge and a handler block, shouldn't the OverflowCHK opcodes have CanRaiseException as well?